src/server/plugins/engine/helpers.ts

```diff
@@ -82,6 +82,12 @@ export const filesize = (bytes: number) => {
   return Math.max(bytes, 0.1).toFixed(1) + byteUnits[i]
 }
 
-export function hasPreviewPath(path: string) {
-  return path.toLowerCase().startsWith(PREVIEW_PATH_PREFIX)
+export function checkFormStatus(path: string) {
```

Comment:
please suggest a better name if can think of?!

Reply:
We refer to these as ["state"](https://github.com/DEFRA/forms-designer/blob/c42b73144903454392191a097ad1ee975049da19/model/src/form/form-metadata/types.ts#L150)